src/main/webapp/app/shared/subject/subject-dialog.component.html

```diff
@@ -22,7 +22,7 @@
                    [translate]="'managementPortalApp.subject.externalId'">
             </label>
             <input type="text" class="form-control" name="externalId" id="field_externalId"
-                   [(ngModel)]="subject.externalId"
+                   [(ngModel)]="subject.externalId"    [disabled]="!!subject.id"
```

Comment:
![image](https://github.com/UoM-Digital-Health-Software/ConnectManagementPortal/assets/9347996/98b89165-9df9-4d57-91ef-00e9cc77ccfc)
I am using the developing mode, the already there internal data cannot be edited anymore? And I found there are no way to delete a subject?